Makefile

```diff
@@ -16,7 +16,8 @@ ldflags := $(LDFLAGS)
 ldflags += -X github.com/cosmos/cosmos-sdk/version.Name=Noble \
 	-X github.com/cosmos/cosmos-sdk/version.AppName=nobled \
 	-X github.com/cosmos/cosmos-sdk/version.Version=$(VERSION) \
-	-X github.com/cosmos/cosmos-sdk/version.Commit=$(COMMIT)
+	-X github.com/cosmos/cosmos-sdk/version.Commit=$(COMMIT) \
+	-X github.com/cosmos/cosmos-sdk/version.BuildTags='ledger'
```

Comment:
In theory aren't we also missing the `netgo` flags? @boojamya

Wondering if we should also bring the more robust way of adding the Ledger flag back ... ensuring system etc 🤔